packages/bpk-component-graphic-promo/src/BpkGraphicPromo.js

```diff
@@ -0,0 +1,202 @@
+    textAlign,
+    textColor,
+  } = props;
+
+  // FIXME: Use useCallback() here when React is updated.
+  const onClickWrapper = (event: React.MouseEvent<HTMLElement>) => {
+    event.stopPropagation();
+    onClick();
+  };
+
+  const cardClasses = [getClassName('bpk-graphic-promo')];
+  if (className) {
+    cardClasses.push(className);
+  }
+
+  const containerClasses = [
+    getClassName('bpk-graphic-promo__container'),
+    getClassName(`bpk-graphic-promo__container--${textAlign}`),
+  ];
+  if (invertVertically) {
+    containerClasses.push(
+      getClassName('bpk-graphic-promo__container--inverted'),
+    );
+  }
+
+  const getTextClasses = (baseClass: string) => {
+    const classes = [getClassName(baseClass)];
+    classes.push(getClassName(`${baseClass}--${textAlign}`));
+
+    return classes.join(' ');
+  };
+
+  return (
+    <BpkCard
+      className={cardClasses.join(' ')}
+      style={{ backgroundImage: `url(${image})`, color: textColor }}
+      onClick={onClickWrapper}
+      aria-label={constructAriaLabel(props)}
+      padded={false}
+    >
+      <div className={containerClasses.join(' ')} aria-hidden>
+        {sponsor && (
+          <div className={getTextClasses('bpk-graphic-promo__sponsor-content')}>
+            <BpkText
+              tagName="span"
+              className={getClassName('bpk-graphic-promo__sponsor-label')}
+            >
+              {sponsor.label}
+            </BpkText>
+            <img
+              className={getClassName('bpk-graphic-promo__sponsor-logo')}
+              alt={sponsor.altText}
+              src={sponsor.logo}
+            />
+          </div>
+        )}
+        <div className={getTextClasses('bpk-graphic-promo__promo-content')}>
+          {!sponsor && kicker && (
+            <BpkText
+              tagName="span"
+              className={getClassName('bpk-graphic-promo__kicker')}
+            >
+              {kicker}
+            </BpkText>
+          )}
+          <BpkText
+            tagName="h2"
+            className={getClassName('bpk-graphic-promo__headline')}
+          >
+            {headline}
+          </BpkText>
+          {strapline && (
+            <BpkText
+              tagName="p"
+              className={getClassName('bpk-graphic-promo__strapline')}
+            >
+              {strapline}
+            </BpkText>
+          )}
+          <BpkButton
+            className={getClassName('bpk-graphic-promo__cta')}
+            onClick={onClickWrapper}
+          >
+            {ctaText}
+          </BpkButton>
+        </div>
+      </div>
+    </BpkCard>
+  );
+};
+
+BpkGraphicPromo.propTypes = {
+  className: PropTypes.string,
+  kicker: PropTypes.string,
+  headline: PropTypes.string.isRequired,
+  strapline: PropTypes.string,
+  image: PropTypes.string.isRequired,
+  sponsor: PropTypes.shape({
+    label: PropTypes.string,
+    logo: PropTypes.string,
+    altText: PropTypes.string,
+  }),
+  ctaText: PropTypes.string.isRequired,
```

Comment:
I would probably rename this to be `buttonText` rather than `ctaText` just for it being cleaner and more obvious on what its called.

Reply:
CTA is the `Call to Action` button. I was thinking that semantically it makes more sense to say `ctaText`, since it is also used in the `aria-label` for the whole component too. It describes the function rather than the type of component. 🙂

Reply:
Changed to `buttonText`. 🙂